helm-charts/falcon-sensor/Chart.yaml

```diff
@@ -15,12 +15,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.17.18
+version: 1.17.19
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 1.17.18
+appVersion: 1.17.19
```

Comment:
Since this requires a specific sensor, I wonder if we should bump to `1.18`?

Reply:
Incoming change